### PR TITLE
Milestone 2: create and remove books

### PR DESCRIPTION
**Pull request by medaminedev66**
- configure the Redux Store and import the necessary methods from Redux.

### CODE REVIEW

File: src/components/Books/Book.js [hunk 1]
```diff
@@ -3,6 +3,11 @@ import PropTypes from 'prop-types';
 import styles from './books.module.css';
 
 class Book extends PureComponent {
+  remove = () => {
+    const { book, removeBook } = this.props;
+    removeBook(book);
+  };
+
```
Comment (dasileker) [inline src/components/Books/Book.js]: - [ ] The Prop for remove is not handling the event properly even with the validation, please fix it

File: src/components/Books/InputBook.js [hunk 2]
```diff
@@ -1,16 +1,44 @@
...
+    title = e.target.value;
+  };
+
+  const addAuthor = (e) => {
+    author = e.target.value;
+  };
+
+  const submitBookToStore = () => {
+    props.addBook(title, author);
+  };
```
Comment (dasileker) [inline src/components/Books/InputBook.js:19]: - [ ] This is not rendering anything, you can make this code more efficient by binding the state

File: src/redux/configureStore.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+import booksReducer from './books/books';
+
+const reducer = combineReducers({
+  booksReducer,
+  // additional reducers could be added here
+});
+
+const store = createStore(reducer, applyMiddleware(logger));
+
+export default store;
```
Comment (dasileker) [inline src/redux/configureStore.js:12]: - [ ] The redux store  the **combineReducer**  doesn't store any data entry, please fix it

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @medaminedev66,** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

- It's a good start but you need to fix some issues in your code, but please always check the console and see if the implemented codes are working to avoid such situations in the future.


Capture d’écran 2021-09-29 182131

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by medaminedev66** [comment]
Hi @dasileker, Thanks for the review. I didn't get the project requirements very well. I will make some changes and make the application works well, and I will submit it again.

**Review by AshakaE — APPROVED** [review]
### Hi @medaminedev66  🙋‍♀️, 

# STATUS APPROVED ✔
Thanks for making the requested changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**